.github/workflows/ci.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       fail-fast: true
       matrix:
         version:
-          - '1.0.5'
+          - '1.0'
```

Comment:
If we actually need to be this specific, let me know.

Reply:
Not anymore. Initially, I was using very accurate versions. But later I decided to assume that all the patch versions will have one precompile folder and the user will always update to the latest patch.